packages/shared/src/UI/components/MiniNetworkSelector/index.tsx

```diff
@@ -41,7 +41,7 @@ const useStyles = makeStyles<{ size: number }>()((theme, props) => ({
 
 export interface NetworkSelectorMinProps {
     selectedChainId: null | ChainId
-    networks: NetworkType[]
+    networks: { ID: string; icon: URL; type: NetworkType }[]
```

Comment:
Use type definition in `@masknet/plugin-infra`

Reply:
masknet/plugin-infra also depend masknet/shared.